python_modules/dagster/dagster/scheduler/scheduler.py

```diff
@@ -362,8 +385,16 @@ def _create_scheduler_run(
             known_state=None,
         )
         execution_plan_snapshot = external_execution_plan.execution_plan_snapshot
-    except Exception:
-        execution_plan_errors.append(serializable_error_info_from_exc_info(sys.exc_info()))
+    except DagsterUserCodeProcessError as e:
+        # is this the right allowlist of exceptions that should still result in creating
+        # a run??
+        if (
+            len(e.user_code_process_error_infos) > 0
+            and e.user_code_process_error_infos[0].cls_name == "DagsterInvalidConfigError"
+        ):
+            execution_plan_errors.append(e.user_code_process_error_infos[0])
```

Comment:
i still have this question - not sure if there's a better way to distinguish between transient errors (where we should retry) and config errors where we shouldn't. Or maybe we should just treat a failed execution plan as a failed tick rather than creating a failed run? I think that feature existed way back in the day when we had no schedule timelines or daemon logs, and the only way to introspect a failed schedule was to either reach into a crontab or do something like this